sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/models/_conversation_analysis_client_enums.py

```diff
@@ -40,6 +115,76 @@ class InnerErrorCode(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     AZURE_COGNITIVE_SEARCH_NOT_FOUND = "AzureCognitiveSearchNotFound"
     AZURE_COGNITIVE_SEARCH_THROTTLING = "AzureCognitiveSearchThrottling"
     EXTRACTION_FAILURE = "ExtractionFailure"
+    INVALID_REQUEST_BODY_FORMAT = "InvalidRequestBodyFormat"
+    EMPTY_REQUEST = "EmptyRequest"
+    MISSING_INPUT_DOCUMENTS = "MissingInputDocuments"
+    INVALID_DOCUMENT = "InvalidDocument"
+    MODEL_VERSION_INCORRECT = "ModelVersionIncorrect"
+    INVALID_DOCUMENT_BATCH = "InvalidDocumentBatch"
+    UNSUPPORTED_LANGUAGE_CODE = "UnsupportedLanguageCode"
+    INVALID_COUNTRY_HINT = "InvalidCountryHint"
+
+class LengthUnit(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+    """The length Unit of measurement
+    """
+
+    UNSPECIFIED = "Unspecified"
+    KILOMETER = "Kilometer"
+    HECTOMETER = "Hectometer"
+    DECAMETER = "Decameter"
+    METER = "Meter"
+    DECIMETER = "Decimeter"
+    CENTIMETER = "Centimeter"
+    MILLIMETER = "Millimeter"
+    MICROMETER = "Micrometer"
+    NANOMETER = "Nanometer"
+    PICOMETER = "Picometer"
+    MILE = "Mile"
+    YARD = "Yard"
+    INCH = "Inch"
+    FOOT = "Foot"
+    LIGHT_YEAR = "LightYear"
+    PT = "Pt"
+
+class Modality(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+    """The modality (format) of ConversationItem (e.g., Text, Transcript)
```

Comment:
The comment has "test, transcript", but the values only have "TEXT"?

Reply:
for this release, they're only supporting text
but i think the April/May release, they're going to support transcript